deploy-scripts/deploy_snapshot.sh

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env bash
-. $(dirname $0)/deploy_common.sh
```

Comment:
are you sure these are not needed?

Reply:
It's a duplication. It's already included in `deploy.sh`.

Reply:
https://github.com/47degrees/hood/blob/fix-publication/deploy-scripts/deploy.sh#L2